src/components/AppLayout/index.tsx

```diff
@@ -102,13 +123,21 @@ const Layout: React.FC<Props> = ({
     path: [SAFE_ROUTES.SETTINGS, WELCOME_ROUTE],
   })
 
+  const onSidebarClick = useCallback(
+    (e: React.MouseEvent): void => {
+      e.stopPropagation()
+      setExpanded((prev) => !prev)
+    },
+    [setExpanded],
+  )
+
   return (
-    <Container>
+    <Container onClick={() => setExpanded(false)}>
       <HeaderWrapper>
         <Header />
       </HeaderWrapper>
       <BodyWrapper>
-        <SidebarWrapper data-testid="sidebar">
+        <SidebarWrapper data-testid="sidebar" $expanded={expanded} onClick={onSidebarClick}>
```

Comment:
What is the dollar sign syntax? 🧐

Reply:
It prevents the attribute from being passed to the DOM.

Reply:
Uau! Thanks for the explanation!

Reply:
https://styled-components.com/docs/api#transient-props – it works w/o a dollar sign too tho.